recipes/pyrost/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set name = "pyrost" %}
+{% set version = "0.1.12" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 7f289093c3b7f8b58bb5a2ecb499275f78bf3704ee9b5b347b2072c9df156ddb
+
+build:
+  number: 0
+  skip: True  # [win]
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - llvm-openmp  # [osx]
+    - libgomp  # [linux]
+  host:
+    - libblas
+    - python
+    - pip
+    - cython
+    - gsl
+    - h5py
+    - numpy
+    - scipy
+  run:
+    - {{ pin_compatible('numpy') }}
+    - python
+
+test:
+  imports:
+    - pyrost
+  requires:
+    - pytest
+    - pip
+    - scipy
```

Comment:
`scipy` and `h5py` should not be here. It seems `scipy` is still required in run, but `h5py` should be linked automatically.

Reply:
I tried, apparently h5py is also required in run, it didn't pass the test.

Reply:
Sorry for the confusion.